crates/rolldown/tests/esbuild/importstar/re_export_star_as_iife_no_bundle/diff.md

```diff
@@ -20,6 +20,7 @@ var mod = (function(exports, foo) {
 "use strict";
 const out = foo;
 
+
```

Comment:
https://hyrious.me/esbuild-repl/?version=0.23.0&b=e%00a.js%00export+*+as+out+from+%22foo%22&b=%00b.js%00export+default+function%28%29+%7B%7D%0A&o=%7B%0A++treeShaking%3A+true%2C%0A%22bundle%22%3A+true%2C%0Aformat%3A+%22iife%22%2C%0A%22external%22%3A+%5B%22*%22%5D%0A%7D, It seems our iife impl is different from esbuild, so even the `__require` runtime is introduced but it is not used at all.